src/main/java/com/code972/elasticsearch/plugins/DictReceiver.java

```diff
@@ -12,25 +12,24 @@
  */
 public class DictReceiver {
     private static String home = System.getProperty("user.home");
-    private static String[] gZipFilePaths = {"plugins/analysis-hebrew/dictH.dict", "/var/lib/hebmorph/dictH.dict", home + "/hebmorph/dictH.dict"};
-    private static String[] hspellFilePaths = {"plugins/analysis-hebrew/hspell-data-files/", "/var/lib/hspell-data-files/", home + "/hspell-data-files/"};
-    private static String DEFAULT_HSPELL_PREFIX = HSpellLoader.PREFIX_NOH;
+    private static String[] filePaths = {"plugins/analysis-hebrew/dictionary.dict", "/var/lib/hebmorph/dictionary.dict", home + "/hebmorph/dictionary.dict", 
+                                            "plugins/analysis-hebrew/hspell-data-files/", "/var/lib/hspell-data-files/", home + "/hspell-data-files/"};
     private static DictHebMorph dict = null;
 
     public static DictHebMorph getDictionary() {
         if (dict == null) {
-            dict = setDictionary();
+            dict = setDefaultDictionary();
         }
         return dict;
     }
 
-    public static boolean setHebmorphDictionary(String dictheb_path) {
-        if (dictheb_path != null) {
-            File file = new File(dictheb_path);
+    public static boolean setDictionary(String path) {
+        if (path != null) {
+            File file = new File(path);
             if (file.exists()) {
                 try {
-                    System.out.println("Successfully loaded from: " + dictheb_path);
-                    dict = DictionaryLoader.loadDicAndPrefixesFromGzip(dictheb_path);
+                    dict = DictionaryLoader.loadDictFromPath(path);
+                    System.out.println("Successfully loaded from: " + path);
```

Comment:
I just want to make sure we can see the loading status in the logs,
successful or not
On Mar 25, 2015 3:14 PM, "EgozyN" <notifications@github.com> wrote:

> In src/main/java/com/code972/elasticsearch/plugins/DictReceiver.java
> <https://github.com/synhershko/elasticsearch-analysis-hebrew/pull/8#discussion_r27123540>
> :
>
> >              if (file.exists()) {
> >                  try {
> > -                    System.out.println("Successfully loaded from: " + dictheb_path);
> > -                    dict = DictionaryLoader.loadDicAndPrefixesFromGzip(dictheb_path);
> > +                    dict = DictionaryLoader.loadDictFromPath(path);
> > +                    System.out.println("Successfully loaded from: " + path);
>
> Those prints aren't needed anyway, any reason to log it? It throws if it
> can't load the plugin anyway (elasticsearch keeps running)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/synhershko/elasticsearch-analysis-hebrew/pull/8/files#r27123540>
> .
>